install.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+if [[ $(whoami) != "root" ]]; then
+  echo "This file needs to be executed as root or with sudo."
+  exit
+fi
+
+defaultdir="/usr/bin"
+
+echo "Directory to install to (default: $defaultdir)"
+read installdir
+
+if [[ $installdir = "" ]]; then
+  installdir=$defaultdir
+fi
+
+if [[ ! -d $installdir ]]; then
+  echo "This directory does not exist."
+  exit
+fi
+
+cd src
+zip -r ../pridefetch.zip *
+cd ..
+echo "#!/usr/bin/env python" | cat - pridefetch.zip > pridefetch
+cp pridefetch "$installdir/"
```

Comment:
This portion of the script leaves pridefetch and pridefetch.zip inside the root of the git repo; should we remove them or use a directory under /tmp?